website/site/layouts/_default/single.html

```diff
@@ -7,22 +7,33 @@
 			<nav class="docs-nav" id="docs-nav">
 				{{ $subjects := where .Site.Pages "Section" "docs" }}
 				{{ range sort $subjects "Params.position" }}
+					{{- if ne .Type "widget" -}}
 					<a href="{{ .Permalink }}" class="nav-link {{ if eq .Params.position $activePage }}active{{ end }}">{{ .Title }}</a>
+					{{- end -}}
 				{{ end }}
 			</nav>
 			<div class="mobile docs-nav">
 				<select class="btn-primary" id="mobile-docs-nav">
 					<option value="" selected="selected">Select A Topic</option>
 					{{ $subjects := where .Site.Pages "Section" "docs" }}
 					{{ range sort $subjects "Params.position" }}
+						{{- if ne .Type "widget" -}}
 						<option value="{{ .Permalink }}" class="nav-link {{ if eq .Params.position $activePage }}active{{ end }}">{{ .Title }}</option>
+						{{- end -}}
 					{{ end }}
 				</select>
 			</div>
 		</aside>
 		<article class="docs-content" id="docs-content">
+			{{- if eq .Title "Widgets V2" -}} <!-- Check if is widgets page, if so, add the partial "widgets" for the widget cloud functionality -->
+				{{ partial "edit-link" . }}
+				<h1 id="widgets">Widgets</h1>
+				{{ .Content }}
+				{{- partial "widgets" . -}} 
```

Comment:
We may need to change this later if we want to have content after the widgets partial. Down the line, we may want to abstract this whole tag-cloud pattern to a shortcode we can add anywhere, to any doc. Totally fine for now, though.

Reply:
Alright! I'm a bit clueless with Hugo's shortcodes, but I'm up for the challenge! But I honestly don't know if there's any way to split a content's markdown... We could create two widgets.md files - one for the top content and one for the bottom. Idk, let me know what you think, @verythorough  :D